packages/video_player/video_player/lib/video_player.dart

```diff
@@ -348,9 +348,11 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         break;
     }
 
-    if (videoPlayerOptions?.mixWithOthers != null) {
-      await _videoPlayerPlatform
-          .setMixWithOthers(videoPlayerOptions!.mixWithOthers);
+    if (videoPlayerOptions == null) {
+      await _videoPlayerPlatform.setMixWithOthers(false);
+    } else {
+      await _videoPlayerPlatform.setAmbientMix(
```

Comment:
Is this `setAmbientMix` going to deprecate `setMixWithOthers`? or are they supposed to work in tandem? A link to the relevant documentation for the APIs that this is implementing would be nice!

Reply:
Yes, it is going to deprecate `setMixWithOthers` at some point. 

- [x] [Voice Prompt Mode](https://developer.apple.com/documentation/avfaudio/avaudiosession/mode/2962803-voiceprompt)
- [x] [AVAudioSessionCategoryAmbient](https://developer.apple.com/documentation/avfaudio/avaudiosessioncategoryambient)